packages/DisplayHeading/__tests__/DisplayHeading.spec.jsx

```diff
@@ -3,8 +3,6 @@ import { mount, render } from 'enzyme'
 
 import DisplayHeading from '../DisplayHeading'
 
-import mockMatchMedia from '../../../config/jest/__mocks__/matchMedia'
-
 describe('DisplayHeading', () => {
   const doMount = (props = {}) => {
     const heading = mount(<DisplayHeading {...props}>Great Deals</DisplayHeading>)
```

Comment:
Same comment as for the Box tests. I think the only reason we used `mount` here was to account for the presence of the `Responsive` component. Without it, we can probably use `shallow` for faster tests.

Please look into switching this to `shallow`.